docs/datamodel/constraints.rst

```diff
@@ -318,6 +318,97 @@ using arbitrary EdgeQL expressions. The example below uses the built-in
     );
   }
 
+
+Constraints and object inheritence
+----------------------------------
+
+If you define a constraint on a parent type, the constraint will apply to all
```

Comment:
I keep reading apply to all as "apply to each one" which feels like delegated. (Like how "I will give an apple to all the children" doesn't feel like they are sharing a single apple) Maybe "the constraint will apply irrespective of the types that extend it" or "A constraint on a parent type is evaluated once globally regardless of how many types extend it"?